components/compositing/compositor.rs

```diff
@@ -235,6 +235,9 @@ pub(crate) struct PipelineDetails {
     /// Whether there are animation callbacks
     pub animation_callbacks_running: bool,
 
+    /// Whether there are animated image
+    pub animated_image_running: bool,
+
```

Comment:
I'm not sure the compositor should care if an animation is a CSS animation or an animated GIF. Can this be integrated into `animations_running` in the `ScriptThread`?